armory/scenarios/poisoning_sleeper_agent.py

```diff
@@ -154,14 +154,14 @@ def poison_dataset(self):
             (
                 self.x_poison,
                 self.y_poison,
-                self.poison_index,
+                poison_index,
```

Comment:
if you set `self.x_poison, self.y_poison, _` instead it will be clearer that the third output isn't used

Reply:
Didn't see this before making the next commit.  See if you are fine with that, otherwise I can use _

Reply:
Didn't see this before making the next commit.  See if you are fine with that, otherwise I can use _

Reply:
I'd use `_` so it's clear that the third output isn't used

Reply:
to clarify, I'd still keep the real variable we care about defined as `poison_index` rather than `true_poison_index`

Reply:
check the latest